### ISSUE TITLE
A PyTorch version of TCLP

### ISSUE THREAD

**Issue by lteyoungjun** [kaist-dmlab/TCLP#5]
Hello author,
Thank you for the release of the code of your paper:) I really love your work.

I really wanna run your code in our pipeline, but the code of TCLP is based on Tensorflow 2.0.
If you don't mind, could you provide PyTorch version?

Thank you in advance ^_^

**Comment by YoojuShin** [comment]
There is no plan for PyTorch. Except the source code related to model, plateau model and label propgation is based on Python only.
